ourney/src/main/java/com/lubycon/ourney/domains/trip/dto/SaveTripRequest.java

```diff
@@ -0,0 +1,15 @@
+package com.lubycon.ourney.domains.trip.dto;
+
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+import java.util.*;
+
+@RequiredArgsConstructor
+@Data
+public class SaveTripRequest {
+    private String tripNm;
+    private Long ownerId;
+    private Date startDate;
```

Comment:
```suggestion
    private LocalDate startDate;
```
Date는 가변 타입이기 때문에 자바8부터 추가된 불변 타입 LocalDate를 사용해도 좋을 것 같아용!

https://namocom.tistory.com/673

Reply:
아 요기는 여행일정이라 나중에 수정도 할수 있게 할라고욤!! 여행 경비 내역 입력시에는 LocalDate로 꼭 써야겠어요!!

Reply:
음음 여행 일정 자체를 수정할 수 있어도 요 객체가 요청 DTO이기 때문에 LocalDate를 써도 괜찮을 것 같아용!

Reply:
아하!!!@@ 네 그렇게 해볼게여 감사합니당!